pymbolic/mapper/constant_folder.py

```diff
@@ -100,7 +100,7 @@ class ConstantFoldingMapper(
 
 
 # Yes, map_product incompatible: missing *args, **kwargs
-class CommutativeConstantFoldingMapper(    # type: ignore[misc]
+class CommutativeConstantFoldingMapper(
```

Comment:
Is that comment on top still valid if the ignore can be removed?

Reply:
Removed.